runtime/queries/markdown/spellchecks.scm

```diff
@@ -0,0 +1 @@
+(paragraph) @spell
```

Comment:
neovim added these `@spell` captures to their `highlights.scm` files rather than having separate files for them (here: `spellchecks.scm`). I am new to tree-sitter highlighting, but would expect that in principle such `highlights.scm` files could be shared between projects. The only constraint I can think of is that of the "highlight names" being used, but at least `@spell` is used in both projects :-).

Reply:
That strategy wouldn't work well with the way helix interprets highlight queries: we prevent future stanzas from matching, so if you had a langauge with

```scm
(comment) @spell
(comment) @comment
```

You would only see the highlight for the spell capture.

Plus, highlight queries are run for every render and I expect that spell queries would not need to be run that often.